File: MySqlDemo/app/src/main/java/com/myandroid/davidwong/mysqldemo/MainActivity.java
package com.myandroid.davidwong.mysqldemo;

import android.content.Intent;
import android.os.Handler;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuItem;
import android.view.View;
import android.widget.EditText;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {


    EditText UsernameEt, PasswordEt;
    private static int SPLASH_TIME_OUT = 50000;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent homeIntent = new Intent(MainActivity.this, HomeActivity.class);
                startActivity(homeIntent);
                finish();
            }
        },SPLASH_TIME_OUT);
        UsernameEt = (EditText)findViewById(R.id.etUserName);
        PasswordEt = (EditText)findViewById(R.id.etPassword);
    }
    public void OnLogin(View view) {
        String username = UsernameEt.getText().toString();
        String password = PasswordEt.getText().toString();
        String type = "login";
        BackgroundWorker backgroundWorker = new BackgroundWorker(this);
        //BackgroundWorker backgroundWorker = new BackgroundWorker(getBaseContext());
        backgroundWorker.execute(type, username, password);
    }
    public void OpenReg(View view) {
        startActivity(new Intent(this,Register.class));
    }

}

File: Slide/app/src/main/java/com/myandroid/davidwong/slide/MainActivity.kt
package com.myandroid.davidwong.slide

import android.support.v4.view.ViewPager
import kotlinx.android.synthetic.main.activity_main.*
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.Menu;
import android.view.MenuItem;
import com.myandroid.davidwong.slide.R.id.viewpager

class MainActivity : AppCompatActivity() {


    private val viewPager: ViewPager? = null;
    private lateinit var slideAdapter: SlideAdapter

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setContentView(R.layout.activity_main)

        slideAdapter = SlideAdapter(applicationContext)
        viewpager.adapter = slideAdapter
    }
}
